<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Autoclicker by negjo</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="justLabel2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="justLabel1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max delay</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minDelayInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxDelayInput">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="currentMinDelay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentMaxDelay">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSettings">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="justLabel3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current min delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="justLabel4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current max delay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runStop">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Run/Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="justLabel5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current state</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentState">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setHotkey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set hotkey</string>
    </property>
   </widget>
   <widget class="QLabel" name="justLabel6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current key</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentKey">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Button</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buttonLeft">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buttonMiddle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buttonRight">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="toggle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hold">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Hold</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
